setup.py

```diff
@@ -21,7 +21,7 @@
 setup(
     name='INGRID',
     version_format='{tag}',
```

Comment:
I believe argument 'version_format' would need to be removed since it seems to be setuptools-git-version specific